browser/ntp_background/ntp_p3a_helper_impl.h

```diff
@@ -23,23 +25,29 @@ enum class MetricLogType;
 
 namespace ntp_background_images {
 
-class NTPP3AHelperImpl : public NTPP3AHelper {
+class NTPP3AHelperImpl : public NTPP3AHelper,
+                         NTPBackgroundImagesService::Observer {
```

Comment:
per google c++ code inheritance should be public:
https://google.github.io/styleguide/cppguide.html#Inheritance